edenia/antelope-tools
# Chore/update block explorer urls 1193

### Update footer links

### What does this PR do?

- Resolve #1193 
- Update the footer links of each network when necessary.
- Fix validation to create an account in faucet route.

### Steps to test

1. Check the configuration files
2. Check the changed urls
